ui/src/shared/components/Dygraph.js

```diff
@@ -65,18 +71,26 @@ export default class Dygraph extends Component {
 
     const graphRef = this.graphRef
     const legendRef = this.legendRef
-    let finalLineColors = overrideLineColors
+    const finalLineColors = [...(overrideLineColors || LINE_COLORS)]
+
+    const hashColorDygraphSeries = {}
+    const {length} = finalLineColors
```

Comment:
@cryptoquick Also noticed that if I add more fields to a graph and then refresh, the colors per series change.  Perhaps this has to do with `hasherino` only running on component mount?